strapi/strapi
# feat(content-releases): add findOne endpoint

### What does it do?

- Adds the findOne endpoint for a Release
- Adds the type contracts 
- Reformats all responses to match the spec with data and meta attributes

### Why is it needed?

- For the release details page

### How to test it?

- Make sure you have an existing Release in the database
- Make a request with the id `/content-releases/:id`
- It should return the release with `actions.meta.count`
- It should throw an error if the user does not have permissions to read a release
